packages/database-editor/src/components/ImportWizard/ImportWizard.tsx

```diff
@@ -0,0 +1,142 @@
+import type { DatabaseEditorContext, DatabaseTable } from '@axonivy/database-editor-protocol';
+import { BasicDialogHeader, Button, Dialog, DialogContent, DialogTrigger, Flex, IvyIcon, toast } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useState, type ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+import './ImportWizard.css';
+import { CreationPage, type CreationParameter } from './pages/CreationPage';
+import { DataSourcePage } from './pages/DataSourcePage';
+import { SelectTablesPage } from './pages/TableSelectionPage';
+import { Timeline } from './Timeline';
+
+export type ImportPage = {
+  page: ReactNode;
+  title: string;
+  requiredData: boolean;
+};
+
+export const ImportWizard = ({ context }: { context: DatabaseEditorContext }) => {
+  const { t } = useTranslation();
+  const [selectedDatabase, setSelectedDatabase] = useState<string>();
+  const [selectedTables, setSelectedTables] = useState<Array<DatabaseTable>>([]);
+  const [creationParams, setCreationParams] = useState<Array<CreationParameter>>([]);
+  const [activePage, setActivePage] = useState(0);
+
+  const resetAll = () => {
+    setSelectedDatabase('');
+    setSelectedTables([]);
+    setCreationParams([]);
+  };
```

Comment:
If you do not hold this states inside this component, but split the DialogContent to a new component and move all the states there then you don't need this reset. Because the content is only mounted when the dialog opens, so also the state will be initialized.